bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -1414,118 +1414,137 @@ public String getClientRole() {
     }
 
     /**
-     * Get the keystore type for client. Default is JKS.
+     * Get the keyfile type for client. Default is JKS.
      *
      * @return
      */
-    public String getTLSKeyStoreType() {
-        return getString(TLS_KEYSTORE_TYPE, "JKS");
+    public String getTLSKeyFileType() {
```

Comment:
I think we cannot change all of these names it is public API.
I would prefer to keep old names

Reply:
will revert to old names